nextflow_schema.json

```diff
@@ -383,6 +383,15 @@
                     "help_text": "If no directory path is passed, vcf files will not be annotated by vep.",
                     "fa_icon": "fas fa-folder-open"
                 },
+                "vep_plugin_files": {
```

Comment:
could you add a specific schema for this input. Mainly checking that the files exists.

Reply:
also, don't you need to add this file to option to `nextflow.config`?